README.md

```diff
@@ -1 +1,48 @@
-# To-do-list
\ No newline at end of file
+# E6-Awesome-Book
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome books
+
+In this project, i designed a To-do list site using webpack
+
+## Built With
+
+- Javascript, HTML and CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+- Git clone the repository to your local machine:
+```
+git clone https://github.com/Enekwestanley/Awesome-book.git
```

Comment:
- [ ] Also the link to clone the project is wrong. Kindly change it to that of the current project.